rust-runtime/aws-smithy-http-server/src/add_extension.rs

```diff
@@ -0,0 +1,67 @@
+// this is vendored from tower-http to reduce public dependencies
```

Comment:
Copied from `axum`, but let's revisit this choice: I think it's fair to re-export `tower-http`.

Reply:
Since we are not planning modifications I think its good to just re-export `tower-http`.

Reply:
I am going to change https://github.com/awslabs/smithy-rs/pull/894 and re-export.